VSRAD.Package/Utils/VsEditor.cs

```diff
@@ -63,8 +65,10 @@ public static void OpenFileInEditor(SVsServiceProvider serviceProvider, string p
                 var textManager = serviceProvider.GetService(typeof(SVsTextManager)) as IVsTextManager2;
                 Assumes.Present(textManager);
 
+                dte.ItemOperations.OpenFile(path); // change active document to set active line according to line marker
```

Comment:
Too many OpenFile(). Please restructure the code.

Reply:
Done in f5158ad